eng/versioning/version_client.txt

```diff
@@ -343,6 +343,7 @@ com.azure.resourcemanager:azure-resourcemanager-devspaces;1.0.0-beta.2;1.0.0-bet
 com.azure.resourcemanager:azure-resourcemanager-devtestlabs;1.0.0;1.1.0-beta.1
```

Comment:
We are also missing the ci.yml file related changes that should create a CI pipeline for the new SDK.